projects/cds/src/profiletag/services/profile-tag.injector.ts

```diff
@@ -54,33 +59,16 @@ export class ProfileTagInjector {
 
   track(): Observable<boolean> {
```

Comment:
One idea for the future refactor is to maybe re-arange the methods to match the order in which they are called. This helps when reading the code. This can be done in the same ticket where we split this service to smaller pieces.